CONTRIBUTING.rst

```diff
@@ -29,8 +29,14 @@ For example, here's a JSON object containing data about a talk at PyGotham 2015:
         "language": "eng",
         "recorded": "2015-08-10",
         "related_urls": [
-            "http://example.com/the/slide/deck.pdf",
-            "http://example.com/a/notebook.nb"
+            {
+              "label": "talk slides",
```

Comment:
The simple (un-labelled) URL are still acceptable, right?

Reply:
They have to be valid, is too much work to review all simple URLs to add appropriate labels.